dlog/kimchi/src/verifier.rs

```diff
@@ -531,7 +531,7 @@ where
                                     ChaChaFinal => &index.chacha_comm.as_ref().unwrap()[3],
                                 };
                                 scalars.push(scalar);
-                                commitments.push(col);
+                                commitments.push(c);
```

Comment:
don't prefer 1 character variable name - switch back to `col`

Reply:
it actually previously was `c` so I just reverted since `col` already exists and `c` is in scope only for a few lines

Reply:
would you like `com` better?

Reply:
i would